packages/scorchwrap/src/plugin.js

```diff
@@ -8,18 +8,13 @@
 /** @typedef {import("./types.js").ScorchWrapPluginOptions} ScorchWrapPluginOptions */
 
 const path = require('path')
-const {
-  WebpackError,
-  RuntimeModule,
-  // RuntimeGlobals,
-  // ModuleDependency,
-} = require('webpack')
+const { WebpackError } = require('webpack')
 const { wrapper } = require('./buildtime/wrapper')
 const { generateIdentifierLookup } = require('./buildtime/aa')
 const diag = require('./buildtime/diagnostics')
 const progress = require('./buildtime/progress')
+const { assembleRuntime } = require('./buildtime/assemble')
```

Comment:
(nit) I'd name the file assembleRuntime or runtimeBuilder for immediate clarity that it's not involved in processing the app sources in any way. But I'm not sure if a person who knows what runtime means in the contect of webpack would not consider it obvious by that point.

Reply:
how about "build"? I like keeping files to a single word when possible, not by any means of course